core/admin/mailu/models.py

```diff
@@ -261,14 +261,21 @@ def resolve_domain(cls, email):
     @classmethod
     def resolve_destination(cls, localpart, domain_name, ignore_forward_keep=False):
         localpart_stripped = None
+        stripped_alias = None
+
         if os.environ.get('RECIPIENT_DELIMITER') in localpart:
             localpart_stripped = localpart.rsplit(os.environ.get('RECIPIENT_DELIMITER'), 1)[0]
 
-        alias = Alias.resolve(localpart, domain_name)
-        if not alias and localpart_stripped:
-            alias = Alias.resolve(localpart_stripped, domain_name)
-        if alias:
-            return alias.destination
+        pure_alias = Alias.resolve(localpart, domain_name)
+        pure_alias_has_wildcard = pure_alias and '%' in pure_alias.email and pure_alias.wildcard
```

Comment:
Sorry, one last thing for further clean up: 
I think we can get rid of line 270 and use in L273 just `pure_alias.wildcard`

Reply:
Yep, you’re right. If the user clicks 'sql like syntax' without actually using it, it’s probably better the behavior is clear instead of smart — where smart has been = check if `%` is actually used. thx!